app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -105,11 +109,58 @@ class MainFragment : Fragment() {
                     .navigate(MainFragmentDirections.actionMainFragmentToNewProfileFragment())
             }
         }
-
+        currentlyActivePeofile()
         setHasOptionsMenu(true)
         return binding.root
     }
 
+    private fun currentlyActivePeofile() {
+        binding.activeProfile.visibility = readInt("ACTIVE_PROFILE_VISIBILITY")
+        binding.activeName.text = readString("ACTIVE_PROFILE_NAME")
+        binding.endTimeTxt.text = readString("END_TIME_TEXT")
+        binding.vibIconImg.setImageResource(readInt("VIBRATE_STATE_ICON"))
+
+        if (StartAlarm.beginStatus == 1) {
+            binding.activeProfile.visibility = View.VISIBLE
+            writeInt("ACTIVE_PROFILE_VISIBILITY", View.VISIBLE)
+
+            binding.activeName.text = "Active Profile: " + StartAlarm.activeProfile
+            writeString("ACTIVE_PROFILE_NAME", "Active Profile: " + StartAlarm.activeProfile)
+
+            binding.endTimeTxt.text = "End Time: " + StartAlarm.endTime + " hours"
+            writeString("END_TIME_TEXT", "End Time: " + StartAlarm.endTime)
+
+            if (StartAlarm.vibState) {
+                binding.vibIconImg.setImageResource(R.drawable.vibration)
+                writeInt("VIBRATE_STATE_ICON", R.drawable.vibration)
+            } else {
+                binding.vibIconImg.setImageResource(R.drawable.mute)
+                writeInt("VIBRATE_STATE_ICON", R.drawable.mute)
+            }
+        }
+        if (EndAlarm.endStatus == 1) {
+            binding.activeProfile.visibility = View.GONE
+            writeInt("ACTIVE_PROFILE_VISIBILITY", View.GONE)
+        }
+    }
+    fun writeString(key: String, value: String) {
```

Comment:
These functions are supposed to define in a different file as shown in the shared link so that we can access them everywhere in the app.